.eslintrc.js

```diff
@@ -3,14 +3,14 @@ module.exports = {
   "env": {
     "mocha": true,
     "node": true,
+    "browser": true,
   },
   "plugins": ["chai-friendly"],
   "rules": {
-    "no-unused-expressions": 0,
```

Comment:
This line can stay. 
`"chai-friendly/no-unused-expressions": 2,` extends `no-unused-expressions`